lib/client-assets.php

```diff
@@ -1030,7 +1030,7 @@ function gutenberg_get_available_image_sizes() {
 function gutenberg_editor_scripts_and_styles( $hook ) {
 	$is_demo = isset( $_GET['gutenberg-demo'] );
 
-	global $wp_scripts;
+	global $wp_scripts, $wp_meta_boxes;
```

Comment:
Actually, the part I've added to disable custom fields is already in Core since branch 5.0. So it's more an update into Gutenberg to make sure dealing with custom fields is done the same way than in Core.

see https://core.trac.wordpress.org/browser/branches/5.0/src/wp-admin/edit-form-blocks.php#L364

Reply:
Right, I see it now. Thanks for explanation 👍